### PR TITLE
Auto-fix linting issues

### PR DESCRIPTION
**Pull request by thomasleese**
These were missed in https://github.com/alphagov/support/pull/669.

### CODE REVIEW

File: config/initializers/inflections.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #   inflect.uncountable %w( fish sheep )
 # end
 ActiveSupport::Inflector.inflections(:en) do |inflect|
-  inflect.uncountable %w( feedback )
+  inflect.uncountable %w(feedback)
```
Comment (AlanGabbianelli) [inline config/initializers/inflections.rb]: This is inconsistent with `%w[string]` that we use elsewhere. We should probably use one or the other, not a mix of both.

Comment (thomasleese) [reply config/initializers/inflections.rb]: Hmm, I personally use `%w()` rather than square-brackets.

File: config/environments/production.rb [hunk 2]
```diff
@@ -52,7 +52,7 @@
   config.log_level = :info
 
   # Prepend all log lines with the following tags.
-  config.log_tags = [ :request_id ]
+  config.log_tags = [:request_id]
```
Comment (AlanGabbianelli) [inline config/environments/production.rb]: Shouldn't we use %i[request_id] here?

File: config/initializers/inflections.rb (hunk 1 above)
Comment (AlanGabbianelli) [reply config/initializers/inflections.rb]: That's fine, I don't mind which one, to be honest. I just think it should be consistent :)

**Pull request merged**